main.go

```diff
@@ -1,94 +0,0 @@
-// time-stamp microservice receives timestamps and returns json form of natural/unix timestamp.
-// Exercise by Free Code Camp.
-package main
-
-import (
-	"encoding/json"
-	"fmt"
-	"net/http"
-	"os"
-	"strconv"
-	"strings"
-	"time"
-)
-
-// Date struct returned as json.
-type Date struct {
-	Unix    *int64  `json:"unix"`
-	Natural *string `json:"natural"`
-}
-
-func main() {
-	http.HandleFunc("/", timestamp)
-
-	server := http.Server{
-		Addr: ":" + os.Getenv("PORT"),
-	}
-	fmt.Println("Listening on [", server.Addr, "]...")
-	err := server.ListenAndServe()
```

Comment:
`log.Fatal(server.ListenAndServe())` would remove the need for the error check below.